gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/schema/LayerConverter.java

```diff
@@ -40,12 +43,25 @@ static Layer toLayer(String id, LayerNestedObject obj) {
       String formId = obj.getForms().keySet().iterator().next();
       layer.setForm(FormConverter.toForm(formId, obj.getForms().get(formId)));
       if (obj.getContributorsCanAdd() != null) {
-        layer.setContributorsCanAdd(ImmutableList.copyOf(obj.getContributorsCanAdd()));
+        ImmutableList<FeatureType> featureTypes =
+            stream(obj.getContributorsCanAdd())
+                .map(LayerConverter::toFeatureType)
+                .collect(toImmutableList());
+        layer.setContributorsCanAdd(featureTypes);
       }
     }
     return layer.build();
   }
 
+  private static FeatureType toFeatureType(String stringValue) {
+    switch (stringValue) {
+      case "points":
+        return FeatureType.POINT;
+      default:
+        return FeatureType.UNKNOWN;
+    }
```

Comment:
Same as above

Reply:
See previous comment. If you feel strongly about it please let's discuss!